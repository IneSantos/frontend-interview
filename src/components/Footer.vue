
<template>
    <footer>
        <div class="site-footer">
           <div class="">
                <h4>About page</h4>
                <p>Play tic tac toe or 4-in-a-row anytime!</p>
                <p>Created by @yourname</p>
            </div>
           <div class="">
                <h4>Contacts</h4>
                <a href="tel:+351123456789">(+351) 123 456 789</a>
                <p>Rua SÃ¡ da Bandira, 111, Porto</p>
            </div>
            <div class="socials">
                <h4>Stay in touch</h4>
                <div>
                    <a href="#"><img src="@/assets/images/facebook.svg" alt="Facebook Logo" /></a>
                    <a href="#"><img src="@/assets/images/twitter.svg" alt="Twitter Logo" /></a>
                    <a href="#"><img src="@/assets/images/instagram.svg" alt="Instagram Logo" /></a>
                </div>
                <div class="subscribe">
                    <input type="email" placeholder="Subscribe our games">
                    <button id="button-addon1" type="submit" class="send-link"><img src="@/assets/images/send.svg" alt="Instagram Logo" /></button>
                </div>
            </div>
        </div>
    </footer>
</template>


<script>
import '@/styles/footer.scss';

export default {
  name: 'Footer'
}
</script>