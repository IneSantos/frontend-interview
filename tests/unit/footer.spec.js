import { shallowMount } from '@vue/test-utils'
import Footer from '@/components/Footer.vue'

describe('Footer.vue', () => {
  it('renders default', () => {
    const wrapper = shallowMount(Footer);
    expect(wrapper).toMatchSnapshot();
  })
})
