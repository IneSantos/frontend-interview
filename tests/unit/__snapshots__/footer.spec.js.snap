// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue renders default 1`] = `
<footer>
  <div class="site-footer">
    <div>
      <h4 class="title">About page</h4>
      <p class="description">Play tic tac toe or 4-in-a-row anytime!</p>
      <p class="description">Created by @yourname</p>
    </div>
    <div>
      <h4 class="title">Contacts</h4> <a href="tel:+351123456789" class="description">(+351) 123 456 789</a>
      <p class="description">Rua SÃ¡ da Bandira, 111, Porto</p>
    </div>
    <div class="socials">
      <h4 class="title">Stay in touch</h4>
      <div><a href="#"><img src="@/assets/images/facebook.svg" alt="Facebook Logo"></a> <a href="#"><img src="@/assets/images/twitter.svg" alt="Twitter Logo"></a> <a href="#"><img src="@/assets/images/instagram.svg" alt="Instagram Logo"></a></div>
      <div class="subscribe"><input type="email" placeholder=" Subscribe our games"> <button type="submit" class="send-link"><img src="@/assets/images/send.svg" alt="Instagram Logo"></button></div>
    </div>
  </div>
</footer>
`;
